cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -453,6 +453,23 @@ public Set<Integer> removedNodes() {
         return nodes;
     }
 
+    /**
+     * Generates list of Kafka node IDs that are going to be added to the Kafka cluster as brokers.
+     *
+     * @return  Set of Kafka node IDs which are going to be added as brokers.
+     */
+    public Set<Integer> addedBrokerNodes() {
```

Comment:
Should it be `scaleUpNodes` (`scaleUpBrokerNodes`) or `addedBrokerNodes`? The naming seems to be inconsistent. Also, are those new nodes or only those scaled-up?

Reply:
> are those new nodes or only those scaled-up?

It will report all nodes on creation as well as newly added brokers on scale up, for this reason I called it `addedBrokerNodes`.

Reply:
Might be worth explaining in the Javadoc.